src/components/ScholarshipDetailCard.jsx

```diff
@@ -122,7 +126,7 @@ export default function ScholarshipDetailCard({ scholarship }) {
   };
 
   return (
-    <Box>
+    <Box className={classes.detailSection}>
```

Comment:
Have you considered using the [**Card**](https://material-ui.com/components/cards/) or [**Paper**](https://material-ui.com/components/paper/) component?